util/mkinstallvars.pl

```diff
@@ -151,3 +157,17 @@ package OpenSSL::safe::installdata;
 
 1;
 _____
+
+######## Helpers
+
+sub _pairs (@) {
```

Comment:
It would be nice to add a comment here describing this function.

Reply:
Added